docs/configuration.rst

```diff
@@ -83,18 +83,49 @@ Podman
     :undoc-members:
 
 
-.. _linters:
+.. _root_lint:
 
 Lint
 ----
 
-Molecule handles project linting by invoking configurable linters.
+Starting with v3, Molecule handles project linting by invoking and external
+lint commands as exemplified below.
 
-Yaml Lint
-^^^^^^^^^
+The decision to remove the complex linting support was not easily taken as we
+do find it very useful. The issue was that molecule runs on scenarios and
+linting is usually performed at repository level.
+
+It makes little sense to perform linting in more than one place per project.
+Molecule was able to use up to three linters and while it was aimed to flexible
+about them, it ended up creating more confusions to the users. We decided to
+maximize flexibility by just calling an external shell command.
+
+.. code-block:: yaml
+
+    lint: |
+        yamllint .
+        ansible-lint
+        flake8
```

Comment:
Is "env" supported? Worth to mention what are possible options here.
Also from which directory it's executed.